core/utils.py

```diff
@@ -6,8 +6,14 @@
 def valid_filename(s):
     return re.sub(r'(?u)[^-\w.]', '', str(s).strip().replace(' ', '_'))
 
-def generate_md(title, highlights):
+def generate_md(title, highlights, with_headings=True, normal_sorting=True):
         text = '# {}\n\n'.format(title)
+        if normal_sorting:
+            highlights = sorted(highlights, key=(lambda highlights: (highlights[3], highlights[4], highlights[2])), reverse=False) # sort by chapter, reference in chapter and creation time
```

Comment:
Maybe something like these would be more descriptive?

```python
class Highlight:
    def __init__(self, row, heading, created, chapter, ref_in_chapter):
        self.row = row
        self.heading = heading
        self.created = created
        #  and so on ...
```

```python
highlights = sorted(highlights, key=(lambda highlight: (highlight.chapter, highlight.ref_in_chapter, highlight.created)), reverse=False)
```

Reply:
Added class in [452e65](https://github.com/Galarius/pinotate/commit/452e6516882b0eae0fc24777091488fc9d43dcf0).